src/iter.rs

```diff
@@ -0,0 +1,37 @@
+use crate::TwoFloat;
+use num_traits::Zero;
+use std::iter::Sum;
+use std::ops::Add;
```

Comment:
We do have the option to build the crate as `no_std`, so should be using

```suggestion
use core::iter::Sum;
use core::ops::Add;
```

(which reminds me, I need to add the `no_std` case back into the CI)